### ISSUE TITLE
MLDataPattern documentation migration to MLUtils

### ISSUE THREAD

**Issue by rmkn85** [JuliaML/MLUtils.jl#115]
Hi, I see that MLDataPattern.jl has comprehensive documentation:
https://mldatapatternjl.readthedocs.io/en/latest/index.html
But the package itself is said to be migrated to MLUtils.jl
What about the documentation? Is it planned to be migrated?
As it covers, in depth, the functionality that was already migrated...
Thanks!

**Comment by darsnack** [comment]
Yes, we should eventually migrate the docs and add more tutorials about typical use cases.
